server/tests/integration/common/actions/engine.organismes.utils.test.ts

```diff
@@ -0,0 +1,74 @@
+
+describe("Tests des actions  engine utilitaires organismes", () => {
+  describe("isOrganismeFiableForCouple", () => {
+    const UAI_REFERENTIEL = "7722672E";
+    const SIRET_REFERENTIEL = "99370584100099";
+
+    const UAI_REFERENTIEL_FERME = "4422672E";
+    const SIRET_REFERENTIEL_FERME = "44370584100099";
+
+    beforeEach(async () => {
+      // Création d'un organisme dans le référentiel avec un couple fiable
+      await organismesReferentielDb().findOneAndUpdate(
+        { uai: UAI_REFERENTIEL, siret: SIRET_REFERENTIEL, nature: "formateur" },
+        { $set: { lieux_de_formation: [{ uai: UAI_REFERENTIEL }], relations: [] } },
+        { upsert: true, returnDocument: "after" }
+      );
+
+      // Création d'un organisme FERME dans le référentiel avec un couple
+      await organismesReferentielDb().findOneAndUpdate(
+        { uai: UAI_REFERENTIEL_FERME, siret: SIRET_REFERENTIEL_FERME, nature: "formateur" },
+        { $set: { lieux_de_formation: [{ uai: UAI_REFERENTIEL_FERME }], relations: [], etat_administratif: "fermé" } },
+        { upsert: true, returnDocument: "after" }
+      );
+    });
+
+    describe("Vérification des cas de couples non fiables", () => {
+      it("Vérifie qu'on couple sans UAI fourni n'est pas fiable", async () => {
+        const isOrganismeFiable = await isOrganismeFiableForCouple(null, SIRET_REFERENTIEL);
+        assert.equal(isOrganismeFiable, false);
+      });
+
+      it("Vérifie qu'on couple sans SIRET fourni n'est pas fiable", async () => {
+        const isOrganismeFiable = await isOrganismeFiableForCouple(UAI_REFERENTIEL, null);
+        assert.equal(isOrganismeFiable, false);
+      });
+
+      it("Vérifie qu'on couple sans UAI ni SIRET fourni n'est pas fiable", async () => {
+        const isOrganismeFiable = await isOrganismeFiableForCouple(null, null);
+        assert.equal(isOrganismeFiable, false);
+      });
+
+      it("Vérifie qu'on couple avec un UAI non présent dans le référentiel n'est pas fiable", async () => {
+        const UNKNOWN_UAI = "1111111A";
+        const isOrganismeFiable = await isOrganismeFiableForCouple(UNKNOWN_UAI, SIRET_REFERENTIEL);
+        assert.equal(isOrganismeFiable, false);
+      });
+
+      it("Vérifie qu'on couple avec un SIRET non présent dans le référentiel n'est pas fiable", async () => {
+        const UNKNOWN_SIRET = "22222222220099";
+        const isOrganismeFiable = await isOrganismeFiableForCouple(UAI_REFERENTIEL, UNKNOWN_SIRET);
+        assert.equal(isOrganismeFiable, false);
+      });
+
+      it("Vérifie qu'on couple fermé dans le référentiel n'est pas fiable", async () => {
+        const organismeFerme = await organismesReferentielDb().findOne({ uai: UAI_REFERENTIEL_FERME });
+        assert.equal(organismeFerme?.etat_administratif, "fermé");
+
+        const isOrganismeFiable = await isOrganismeFiableForCouple(UAI_REFERENTIEL_FERME, SIRET_REFERENTIEL_FERME);
+        assert.equal(isOrganismeFiable, false);
+      });
+    });
```

Comment:
On a pas mal de tests ici qui ont la même structure, cad seules les données d'entrée et le résultat changent.
Je pense qu'on pourrait améliorer la lisibilité en extrayant les données de test dans une structure séparée, et puis d'itérer dessus pour exécuter les tests, un peu comme j'ai fait avec les permissions.
Cad qqch du genre :
```ts
    const tests = [
      {
        label: "Vérifie qu'on couple sans UAI fourni n'est pas fiable",
        uai: null,
        siret: SIRET_REFERENTIEL,
        expectedFiable: false,
      },
      {
        label: "Vérifie qu'on couple sans SIRET fourni n'est pas fiable",
        uai: UAI_REFERENTIEL,
        siret: null,
        expectedFiable: false,
      },
      {
        label: "Vérifie qu'on couple sans UAI ni SIRET fourni n'est pas fiable",
        uai: null,
        siret: null,
        expectedFiable: false,
      },
      //...
    ];
    tests.forEach((test) => {
      it(test.label, async () => {
        expect(await isOrganismeFiableForCouple(test.uai, test.siret)).toBe(test.expectedFiable);
      });
    });
```
Qu'en penses-tu ?